CodeReview/src/items/VReviewComment.cpp

```diff
@@ -72,6 +72,19 @@ void VReviewComment::initializeForms()
 	addForm(container);
 }
 
+Visualization::Item::UpdateType VReviewComment::needsUpdate()
+{
+	return Visualization::Item::StandardUpdate;
+}
+
+void VReviewComment::determineChildren()
+{
+	Super::determineChildren();
+
+	// TODO implement mechanism for time-based updates to avoid exploiting determineChildren and needsUpdate
```

Comment:
Well, actually I think it's rather ok to do it this way, at least until we see a performance issue. So please remove this TODO.